test-files/cmd/slow/MgcpTranslationTest.sh

```diff
@@ -22,6 +22,9 @@ compareFiles test-output/cmd/slow/translation/MgcpTranslated.osm test-files/tran
 #
 # Mattj Oct 13
 
+# LOTS of debug output
+#CONFIG="-D ogr.debug.dumptags=true -D ogr.debug.lookupcolumn=true -D ogr.debug.lookupclash=true -D ogr.debug.dumpvalidate=true"
```

Comment:
Added so that I don't have to copy it in each time I need more debug output.
I might make this a `*.conf` file